projects/greedy_infomax/experiments/__init__.py

```diff
@@ -23,9 +23,19 @@
 from .sigopt_experiments import CONFIGS as SIGOPT_EXPERIMENTS
 from .small_sparse import CONFIGS as SMALL_SPARSE
 from .sparse_resnets import CONFIGS as SPARSE_RESNETS
+from .block_wise_training import CONFIGS as BLOCK_WISE
```

Comment:
(low priority) Run `isort` on this file to alphabetize imports.

Reply:
Done